go.mod

```diff
@@ -9,7 +9,7 @@ require (
 	github.com/cloudflare/cfssl v0.0.0-20180323000720-5d63dbd981b5
 	github.com/container-storage-interface/spec v1.2.0
 	github.com/docker/distribution v2.8.1+incompatible
-	github.com/docker/docker v23.0.3+incompatible
+	github.com/docker/docker v24.0.0-rc.2.0.20230528104423-2ebd97dec1a2+incompatible
```

Comment:
Perhaps you can

- add a comment to indicate that this is "25.0-dev" to prevent confusion `// master (v25.0.0-dev)`
- add a commit to update the image-spec to rc3
- update moby to current master (which brings in https://github.com/moby/moby/pull/45437)

Reply:
done